packages/ndla-ui/src/ProgrammeCard/ProgrammeCard.tsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React from 'react';
+import styled from '@emotion/styled';
+import SafeLink from '@ndla/safelink';
+import { spacing, colors, misc, breakpoints, mq } from '@ndla/core';
+
+interface StyledImage {
+  isMobile: boolean;
+}
+interface Image {
+  src: string;
+  alt: string;
+}
+export interface Programme {
+  id: string;
+  programmeTitle: {
+    title: string;
+    language: string;
+  };
+  programmeImgDesk: Image;
+  programmeImgMob: Image;
```

Comment:
Ser ikkje heilt kvifor du trenger programme i disse verdiene i og med at dei er den del av Programme-komponent. Også lurt med tydelige navn.
```suggestion
  title: {
    title: string;
    language: string;
  };
  desktopImage: Image;
  mobileImage: Image;
```

Reply:
Oppdatert

Reply:
For min del holder det at du klikker på resolve conversation i stedet for å svare på alle kommentarer. Blir litt mindre epost på den måten. Alle som har reviewa får uansett melding når du pusher til branch.